src/main/java/com/conveyal/gtfs/validator/ReferencesTripValidator.java

```diff
@@ -65,7 +65,7 @@ public void validateTrip(
         });
         stopAreas.forEach(stopArea -> {
             if (stopArea != null) {
-                referencedLocations.add(stopArea.area_id);
+                referencedStopAreas.add(stopArea.area_id);
```

Comment:
Did you intend to merge `referencedLocations` and `referencedStopAreas`?

Reply:
No, was a typo! In this case `referencedLocations` should definitely be `referencedStopAreas`.

Reply:
Then why is `referencedLocations` still being used (e.g. line 63)